models/staging/tpdm/base/base_tpdm__schools.sql

```diff
@@ -0,0 +1,37 @@
+with schools as (
+    {{ source_edfi3('schools') }}
+),
+renamed as (
```

Comment:
could we add a newline between CTEs?
```suggestion
),

renamed as (
```